cclient/serialization/json/json_serializer.c

```diff
@@ -49,6 +49,44 @@ retcode_t utarray_to_json_array(UT_array* ut, cJSON* json_root,
   return RC_OK;
 }
 
+retcode_t json_array_to_int_array(cJSON* obj, const char* obj_name,
+                                  int_array* in) {
+  cJSON* json_item = cJSON_GetObjectItemCaseSensitive(obj, obj_name);
+  if (cJSON_IsArray(json_item)) {
+    cJSON* current_obj = NULL;
+    int_array_allocate(in, cJSON_GetArraySize(json_item));
+    int* in_arr_p = in->array;
```

Comment:
I think you better use `in->array` directly, because as much a as it seems not probable, if the underlying type is change from int to size_t then you cast to wrong type